src/instructlab/cli/model/evaluate.py

```diff
@@ -135,6 +137,43 @@
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
+    "--input-questions",
+    type=click.Path(exists=True, path_type=pathlib.Path),
+    default=None,
+    help="Path to the questions and reference answers the model will be evaluating in the DK-Bench evaluation",
+)
+@click.option(
+    "--output-file-formats",
+    type=click.STRING,
+    default="jsonl",
+    show_default=True,
+    help="Comma-separated list of file formats for results of the DK-Bench evaluation. Valid options are csv, jsonl, and xlsx. If this option is not provided the results are written as a .jsonl file",
+)
+@click.option(
+    "--system-prompt",
+    type=click.STRING,
+    default=ragas_eval._DEFAULT_SYSTEM_PROMPT,
+    help="Prompt for the model when getting responses in the DK-Bench evaluation",
+)
+@click.option(
+    "--temperature",
+    type=click.FLOAT,
+    default=0.0,
+    help="Temperature for the model when getting responses in the DK-Bench evaluation",
+)
+@click.option(
+    "--model-name",
```

Comment:
Could we please get rid of `--model-name` and instead rely on reading the columns from the dataframe? Otherwise this will introduce greater confusion for the end-user.

Reply:
I'm not sure I follow. The correct model name for an endpoint is needed to get responses from the student model.

 The way I had it before in the PR was to do a model list on the endpoint, if there was more than one model being served at the endpoint I raised an error. If not, then the model's name at the endpoint was set to model.

For models that are local, the model name is what we set it to because we're doing the serving ourselves with `launch_server`. For models that are endpoints, I'm checking the model name with `get_endpoint_model_name()`. Do you have any feedback on this approach?

@danmcp any thoughts here?

Reply:
@alimaredia Based on our earlier conversation, `--model` is the argument that would stay as a constant among all of the evaluations, so if we change the eval from `mt-bench` to `dk-bench`, we would be evaluating the same model against a different benchmark.

If our student model is running on an endpoint, then this should be the behavior currently handled by `MMLU` and `MT-Bench` as well, and therefore supported by this flag.

Reply:
@RobotSail I see, I didn't realize that having only local models was part of the consistency you were envisioning in the first cut. 

After digging into the code a little bit, having every benchmark evaluate endpoints actually shouldn't be that hard to implement but would be outside of the scope of getting DK-Bench initially working. I'm going to go ahead and remove all of the endpoint functionality from this PR.

Reply:
Yeah, I definitely think it's a good change but we need to keep PRs focused in order to make a high volume of changes.